client/components/catalog/Card/index.vue

```diff
@@ -133,22 +146,25 @@ export default {
 .card-body {
   padding: 0.625rem 0 0;
 
-  .v-card__title {
-    line-height: 1.75rem;
+  .header {
+    display: flex;
+    align-items: center;
   }
 
   .schema-name {
-    padding: 0 0 0 0.25rem;
+    flex-grow: 1;
```

Comment:
Also a class for [flex-grow](https://vuetifyjs.com/en/styles/flex/#flex-grow-and-shrink).